AuroProctoringSDK/src/main/java/com/example/auroproctoringsdk/detector/FaceComapre.kt

```diff
@@ -0,0 +1,99 @@
+/*
+package com.example.auroproctoringsdk.detector// snippet-start:[rekognition.kotlin.compare_faces.import]
+import aws.sdk.kotlin.runtime.auth.credentials.StaticCredentialsProvider
+import aws.sdk.kotlin.services.rekognition.RekognitionClient
+import aws.sdk.kotlin.services.rekognition.model.CompareFacesMatch
+import aws.sdk.kotlin.services.rekognition.model.CompareFacesRequest
+import aws.sdk.kotlin.services.rekognition.model.Image
+import aws.smithy.kotlin.runtime.auth.awscredentials.Credentials
+import java.io.File
+import kotlin.system.exitProcess
+// snippet-end:[rekognition.kotlin.compare_faces.import]
+
+*/
+/**
+Before running this Kotlin code example, set up your development environment,
+including your credentials.
+
+For more information, see the following documentation topic:
+https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/setup.html
+ *//*
+
+
+suspend fun faceCompareProcess(args: ArrayList<String>) {
+
+
+
+    val usage = """
+        Usage: <pathSource> <pathTarget>
+
+        Where:
+            pathSource - The path to the source image (for example, C:\AWS\pic1.png). 
+            pathTarget - The path to the target image (for example, C:\AWS\pic2.png). 
+    """
+
+    if (args.size != 2) {
```

Comment:
please add code commenet what this "arg" denote ? and why you have put the thereshold to 70F ?